nebula/ui/components/forms/VPNTextField.qml

```diff
@@ -10,50 +10,69 @@ import Mozilla.VPN 1.0
 import components 0.1
 
 TextField {
-    // TODO Add strings for Accessible.description, Accessible.name
     property bool hasError: false
     property bool showInteractionStates: true
     property bool forceBlurOnOutsidePress: true
     property alias _placeholderText: centeredPlaceholderText.text
+    property string _accessibleName: _placeholderText
+    property string _accessibleDescription: ""
 
     id: textField
 
+    Accessible.name: _accessibleName
+    Accessible.description: _accessibleDescription
+    Accessible.focused: textField.focus
+    Layout.alignment: Qt.AlignVCenter
+    Layout.preferredHeight: VPNTheme.theme.rowHeight
+
     background: VPNInputBackground {
         id: textFieldBackground
     }
-
-    font.pixelSize: VPNTheme.theme.fontSizeSmall
-    font.family: VPNTheme.theme.fontInterFamily
+    bottomPadding: VPNTheme.theme.windowMargin / 2
     color: VPNTheme.colors.input.default.text
+    cursorDelegate: VPNCursorDelegate {}
     echoMode: TextInput.Normal
+    focus: true
+    font.family: VPNTheme.theme.fontInterFamily
+    font.pixelSize: VPNTheme.theme.fontSizeSmall
     inputMethodHints: Qt.ImhNoPredictiveText | Qt.ImhSensitiveData
-    onActiveFocusChanged: if (focus && typeof(vpnFlickable) !== "undefined" && vpnFlickable.ensureVisible) vpnFlickable.ensureVisible(textField)
-    selectByMouse: true
-    Layout.preferredHeight: VPNTheme.theme.rowHeight
-    Layout.alignment: Qt.AlignVCenter
-    verticalAlignment: TextInput.AlignVCenter
-    placeholderTextColor: VPNTheme.colors.grey40
     leftPadding: VPNTheme.theme.windowMargin
+    placeholderTextColor: VPNTheme.colors.grey40
     rightPadding: VPNTheme.theme.windowMargin
+    selectByMouse: true
     topPadding: VPNTheme.theme.windowMargin / 2
-    bottomPadding: VPNTheme.theme.windowMargin / 2
-    focus: true
-    cursorDelegate: VPNCursorDelegate {}
+    verticalAlignment: TextInput.AlignVCenter
+
+    onActiveFocusChanged: if (focus && typeof(vpnFlickable) !== "undefined" && vpnFlickable.ensureVisible) {
+        vpnFlickable.ensureVisible(textField);
+    }
+    // This is a workaround to make VoiceOver on macOS work.
+    // After gaining initial focus or typing in TextField the screen reader
+    // fails to narrae any accessible content and action. After regaining active
+    // focus the screen reader keeps working as expected.
+    onTextChanged: {
+        textField.focus = false;
+        textField.forceActiveFocus();
+    }
```

Comment:
Should we apply this to VPNTextArea as well? And maybe prevent it from being used by VPNPasswordInput?